mobify/progressive-web-scaffold
# Remove duplicate images in wrong place

Some images that were added in both the original `~/app/static` and `~/web/app/static` locations got merged together, resulting in duplicate files in different locations. I've removed the unnecessary ones.

## Changes
- Remove unnecessary duplicate images

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Verify that the Homepage carousel images work as expected.
